.github/workflows/ci.yml

```diff
@@ -65,15 +65,18 @@ jobs:
           - os: ubuntu-latest
             platform: linux
             arch: x86_64
+            toolchain: stable
           - os: macos-latest
             platform: darwin
             arch: x86_64
+            toolchain: stable
           - os: windows-latest
             platform: windows
             arch: x86_64
+            toolchain: stable-x86_64-pc-windows-gnu
```

Comment:
do we have to specify this?

Reply:
yeah otherwise the default on windows is msvc

Reply:
I would recommend using the gnu toolchain on windows, especially if you need to compile a c project, as there are fewer bugs than with msvc.